tests/integration/mutation/create/embeddings/embedding_test.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+//go:build linux
```

Comment:
todo: Please change the way you constrain these tests.  The current setup will prevent me, or any other linux users from running `go test ./...` or even `make test`, this could be a bit of a nuisance.  Perhaps introduce something similar to the `SupportedClients` `TestCase` property.

Reply:
Done